src/rust-crypto/verification.ts

```diff
@@ -40,13 +41,16 @@ export class RustVerificationRequest
     extends TypedEventEmitter<VerificationRequestEvent, VerificationRequestEventHandlerMap>
     implements VerificationRequest
 {
+    /** a reëmitter which relays VerificationRequestEvent.Changed events emitted by the verifier */
+    private readonly _reEmitter: TypedReEmitter<VerificationRequestEvent, VerificationRequestEventHandlerMap>;
```

Comment:
Why underscored?

Reply:
Our code style is explicit here

> Private members are private and not prefixed unless required for naming conflicts.

https://github.com/vector-im/element-web/blob/develop/code_style.md#typescript--javascript

Reply:
yes, my question is do you want me to fix the other properties here or in a separate PR

Reply:
Happy for it to be here, approval stands

Reply:
turns out the other properties *should* have underscores because there are getter methods which conflict